DNN Platform/Library/Entities/Tabs/TabVersions/TabVersion.cs

```diff
@@ -51,6 +51,11 @@ public class TabVersion : BaseEntityInfo
         /// </summary>
         public DateTime TimeStamp { get; set; }
 
+        /// <summary>
+        /// Creation date displayed in current user's preferred timezone
+        /// </summary>
+        public DateTime CreateOnUserLocalDate { get; set; }
```

Comment:
I don't think we want this here. Looks more like a UI-specific issue. Instead of adding a property I would change how the dates are presented on the view.

@dnnsoftware/tag ?

Reply:
There might be use for a function, but not for a property

Reply:
And also this is not used anywhere now.